/**:
  ros__parameters:
    max_hp: 500
    initial_projectiles: 200
